tutorials/quick_start/README.md

```diff
@@ -167,6 +167,15 @@ in the future.
         <td>30.18</td>
         <td>3.4</td>
     </tr>
+    <tr>
+        <td>Pose Estimation</td>
```

Comment:
It can't be seen in the plain text file but if you look at the generated readme you'll see that something is wrong with the table's coloring (here and under "classification" tab which is not necessarily relevant to this PR but if you're fixing the first one maybe address this as well).

Reply:
Won't fix